config/routes.rb

```diff
@@ -11,7 +11,13 @@
     resources :dashboard, only: [:index]
     resources :loans
     resources :organizations
-    resources :project_steps
+    resources :project_steps do
+      collection do
+        delete '', to: 'project_steps#batch_destroy'
+        patch 'adjust_dates', to: 'project_steps#adjust_dates'
+        patch 'finalize', to: 'project_steps#finalize'
+      end
```

Comment:
Please just use `patch` then.

On Thu, Apr 21, 2016 at 12:21 PM, Joseph Elliott <notifications@github.com>
wrote:

> In config/routes.rb
> <https://github.com/sassafrastech/madeline_system/pull/66#discussion_r60611897>
> :
>
> > @@ -11,7 +11,13 @@
> >      resources :dashboard, only: [:index]
> >      resources :loans
> >      resources :organizations
> > -    resources :project_steps
> > +    resources :project_steps do
> > +      collection do
> > +        delete '', to: 'project_steps#batch_destroy'
> > +        patch 'adjust_dates', to: 'project_steps#adjust_dates'
> > +        patch 'finalize', to: 'project_steps#finalize'
> > +      end
>
> delete :batch_destroy didn't work. Is there a cleaner way to map the
> empty subpath to an controller method?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/sassafrastech/madeline_system/pull/66/files/b98c811992a1057a0d8dbaa07fb2a0da5c6b01c0#r60611897>
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org